string.md

```diff
@@ -7,19 +7,22 @@ toc: true
 
 The `STRING` [data type](data-types.html) stores a string of characters.
 
-In CockroachDB, the following are aliases for `STRING` and `STRING(n)`: 
+In CockroachDB, the following are aliases for `STRING`: 
 
 - `CHARACTER`
-- `CHARACTER(n)`
 - `CHAR` 
-- `CHAR(n)` 
 - `VARCHAR`
-- `VARCHAR(n)` 
 - `TEXT`
 
+And the following are aliases for `STRING(n)`:
+
+- `CHARACTER(n)`
+- `CHAR(n)` 
```

Comment:
It looks like `CHAR(n)` and `CHAR VARYING(n)` behave the same: both use `n` as the max limit, but neither add padding if you pass a value below the limit. Do we want `CHAR(n)` to pad with spaces if below the limit?

```
root@:26257> CREATE TABLE strings2 (a STRING PRIMARY KEY, b CHAR VARYING(4), c CHAR(4));
CREATE TABLE
root@:26257> show columns from strings2;
+-------+-----------+-------+---------+
| Field |   Type    | Null  | Default |
+-------+-----------+-------+---------+
| a     | STRING    | false | NULL    |
| b     | STRING(4) | true  | NULL    |
| c     | STRING(4) | true  | NULL    |
+-------+-----------+-------+---------+
root@:26257> insert into strings2 values ('1', '123', '123');
INSERT 1
root@:26257> select * from strings2;
+---+-----+-----+
| a |  b  |  c  |
+---+-----+-----+
| 1 | 123 | 123 |
+---+-----+-----+
```